sudoku/script.js

```diff
@@ -1,8 +1,21 @@
-let div = document.querySelectorAll('div:not(.wrapper)');
+let div = document.querySelectorAll('.board>div');
 let btn = document.querySelector('.change');
+let check = document.querySelector('.check');
+let answers = [];
 for (let i = 0; i < div.length; i++) {
   div[i].classList.add(i + 1);
 }
+let wrapper = document.querySelector('.board');
+wrapper.oninput = (event) => {
+  let regex = /^\d$/;
```

Comment:
Seems like you didn't changed this variable, you can specify it as a constant
```suggestion
  const regex = /^\d$/;
```